### DISCUSSION TITLE
Custom Notification Settings for incoming snaps and chats

### DISCUSSION THREAD

**Discussion by RevealedSoulEven** [rodit/SnapMod#126] [Ideas & Feature Suggestions]
We all know that Snapchat doesn't allow us to select whether we need notification for snaps or chats. It has an option for user-wise to select notification for Snapchat but we cannot Switch between it that we need either chat or either snap notifications. So if you can add feature where we can select that we either need snap notification or chat notification so it will be better because our notification bar is spammed due to "sent a snap" notification. And it's quite awful. 

Referring to my last discussion, I mentioned to add incoming chat messages in notification instead of "sent a message"

So please add some notification tweaks where we can customise whether we need "sent a message" or "sent a snap" notification or not. So we can easily block all "sent s snap" notification and get only incoming messages notifications. 

Hope my point is clear, Thank You!

**Comment by rodit** [comment]
Added notification filter in 1.7.0

**Reply by RevealedSoulEven** [reply]
There is no new releases yet

**Reply by RevealedSoulEven** [reply]
Ahh. Sorry, My Bad.

By the way, it's great😌 Good Job @rodit
